flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/sinks/OrionTestSink.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright 2014 Telefonica Investigación y Desarrollo, S.A.U
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-connectors/commit/fc3ac6b227108efd40148ab1ac465d28d81e896f